e2e_test/e2e_test.go

```diff
@@ -461,6 +483,74 @@ var _ = g.Describe("E2E integration testing", g.Ordered, func() {
 				}, timeout, interval).Should(BeTrue())
 			})
 		})
+
+		g.Context("With one more ServiceFunctionChains than resources", func() {
+			g.It("Should the SFC Pod run after deleting another SFC", func() {
+
+				g.By("Should have initially not SFCs")
+				sfcLs := testutils.SfcList(dpuSideClient, vars.Namespace)
+				Expect(sfcLs.Items).To(BeEmpty())
+
+				g.By("Should initially detect number of DPU resources")
+				dpuResNum := getMaxDpuResources(dpuSideClient)
+				Expect(dpuResNum).To(BeNumerically(">=", 2))
+				sfcNum := (dpuResNum / 2)
+
+				g.By("Should create N SFCs")
+				for i := 0; i < sfcNum; i++ {
+					sfc := sfcNew(i, imageRef)
+					testutils.SfcCreate(dpuSideClient, sfc)
+				}
+
+				g.By("Should get all N SFC Pods running")
+				for i := 0; i < sfcNum; i++ {
```

Comment:
I think the Marvell DPU only supports 1 SFC atm?

Reply:
Yes. On the DPU side, the number `NoOfPortPairs` is hard coded to two (resulting in two `openshift.io/dpu` resources). And since a SFC Pod requires two such resources, only one SFC can be started at a time.

I think that should change, because for Marvell (currently) those resources are just veth pairs. There is no need to limit those resources (I think), or there is no need to keep them not much higher.

Anyway. The test doesn't care. It checks how many resources are available. It works fine with having only 2 resources (1 SFC).